Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -2596,6 +2602,14 @@ private void SetSelectedRows (BocListRow[] selectedRows)
         _selectorControlCheckedState.Add(RowIDProvider.GetItemRowID(row));
     }
 
+    public void ValidationComplete (bool hasValidationErrors)
+    {
+      if (hasValidationErrors)
+      {
+
```

Comment:
why is this empty?
```suggestion
```

Reply:
Leftover code I forgot the remove. Sorry about that.